requirements.txt

```diff
@@ -3,7 +3,7 @@ attrdict==2.0.0
 eventlet==0.19.0
 python-socketio==1.6.1
 numpy==1.13.1
-Pillow==2.2.1
+Pillow==6.2.0
```

Comment:
Not sure it is a good idea. If you upgrade, the configuration will not match the car's configuration and you may end up with non-runnable software at the test site. 

We realized, that keeping all versions AS IS (except ROS) and try install everything natively is the best approach in this project.